src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -225,6 +226,22 @@ EscrowCreate::preflight(PreflightContext const& ctx)
             return temDISABLED;
     }
 
+    if (ctx.tx.isFieldPresent(sfData))
+    {
+        if (!ctx.tx.isFieldPresent(sfFinishFunction))
+        {
+            JLOG(ctx.j.debug())
+                << "EscrowCreate with Data requires FinishFunction";
+            return temMALFORMED;
+        }
+        auto const data = ctx.tx.getFieldVL(sfData);
+        if (data.size() > maxWasmDataLength)
```

Comment:
this condition tested in the HostFunctions, maybe you should add lcov_exclude

Reply:
This is in `EscrowCreate`, so a direct upload - not via the host function.